src/disk_cache.rs

```diff
@@ -0,0 +1,1064 @@
+use bytes::Bytes;
+use bytes::BytesMut;
+use futures::future::BoxFuture;
+use futures::stream;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use object_store::buffered::BufWriter;
+use object_store::Attribute;
```

Comment:
nit: Convention in most of the files is `object_store:: { }`. This also comes up as a IDE / AI suggestion. Please see if changing it would be appropriate here.

Reply:
thank you, i've fixed the import style in af1cd76